app/models/spree/adjustment.rb

```diff
@@ -73,7 +72,7 @@ class Adjustment < ActiveRecord::Base
     scope :additional, -> { where(included: false) }
 
     scope :enterprise_fee, -> { where(originator_type: 'EnterpriseFee') }
-    scope :admin,          -> { where(source_type: nil, originator_type: nil) }
```

Comment:
if both source and adjustable are the same thing, how come now we don't check the adjustable?

Reply:
So the defining characteristics of an "admin" adjustment were that the `source` and `originator` were both `nil`. All other adjustments have an `originator` of a specific type, like `EnterpriseFee` or `TaxRate`. I think in this case the adjustable is always the order, but that isn't _unique_ to admin adjustments (other adjustments can have the order as their adjustable).

Reply:
I'm also a little confused still - so Matt, you're saying that admin adjustments have a non-nil adjustable, and that's why we can't just make this line:

`scope :admin,          -> { where(adjustable_type: nil, originator_type: nil) }`

Right?

My follow up question, then, would be: why aren't we just removing that condition entirely? Like this:

`scope :admin,          -> { where(originator_type: nil) }`

I suppose that might return adjustments that we don't consider admin adjustments? 

I haven't looked through the whole PR yet, so maybe it will become clear once I do. This just caught my attention first.